src/middleware/ErrorMiddleware.js

```diff
@@ -34,29 +28,28 @@ module.exports = function (
       res.format({
         text: () => this.sendTextResponse(err, req, res),
         html: () => this.sendHtmlResponse(err, req, res),
-        json: () => this.sendJsonResponse(err, req, res)
+        json: () => this.sendJsonResponse(err, req, res),
       });
 
       next();
     }
 
     logErrorStack(err) {
-      const statusCode = err.statusCode || 0;
+      const { statusCode } = err;
       const checkStatus = (status) => status === statusCode;
 
       if (!_some(this.EXCLUDE_STATUSCODE_FROM_LOGS, checkStatus)) {
-        Logger.error(err, '\n', err.stack, '\n', (err.data || ''));
```

Comment:
`err.data` is never `undefined`, `logErrorStack` is called after `appendRequestData` which _always_ assigns a `data` value